src/Fable.Import.RemoteDev.fs

```diff
@@ -62,5 +62,8 @@ module RemoteDev =
     [<Emit("window.__REDUX_DEVTOOLS_EXTENSION__.connect($0)")>]
     let connectViaExtension<'msg> (options: Options<'msg>): Connection = jsNative
 
-    [<Import("extractState","remotedev")>]
-    let extractState (x: obj): obj = jsNative
+    [<Import("parse","jsan")>]
+    let parse (x: string): obj = jsNative
```

Comment:
Is `jsan` correct?

What API are you trying to import? If this is `JSON.parse` it should be available under `Fable.Core.JS.JSON.parse`

Reply:
Yes, it is correct. Redux DevTools uses [jsan](https://github.com/kolodny/jsan) for serialization. For this reason remotedev also uses jsan.